designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -1,34 +1,50 @@
-# Policy Mapping Workflow with IP/MAC
+# Multi-Policy Mapping Workflow
 
-This document describes the proposed changes for policy mapping workflow with IPs/MACs.
+This document describes the proposed changes for multi-policy mapping workflow.
 
 ## Background
 
 Currently, we have two ways to determine which policy gets applied to which workload:
 
-- Provide the VNF security manager with encapsulation tag
-- Provide the VNF security manager with IP/MAC
+- Providing the VNF security manager with the encapsulation tag
+- Providing the VNF security manager with the IP/Mac information and a list of polices/tags to be associated
 
 If the manager is supporting [Policy Mapping](https://github.com/opensecuritycontroller/security-mgr-api/blob/master/src/main/java/org/osc/sdk/manager/Constants.java#L70) and [Security Group](https://github.com/opensecuritycontroller/security-mgr-api/blob/master/src/main/java/org/osc/sdk/manager/Constants.java#L60), we provide the manager with the security group name and members (their IPs and MACs) information to bind the policy.
 
+**Policy mapping workflow**
+
+![](./images/policy_mapping.png)
+
+OSC connects to the manager by adding a manager connector and syncs the domains and the policies avaiable with the manager. The security group on OSC contains workload information. As a part of the binding process, OSC provides the security manager with workload IP/MAC information and the list of policies or tags to be associated with the workload. The security manager provides the supplied information to the security appliance. The security appliance applies the right policies to the workload based on the IP/MAC information.
```

Comment:
split this up into bullet or labelled points based on the graphic above.

Also in the diagram, replace "binding" from step 3 with "associate list of policies/tags with Security group workloads"